test/functional/utils/test-bytes.js

```diff
@@ -33,4 +35,22 @@ describe('stringToBytes', function() {
     }).to.throw();
   });
 
+  it('should convert bytes array to string', () => {
+    const str = bytesToString(new Uint8Array([102, 111, 111]));
+    expect(str).to.equal('foo');
+  });
+
+  it('should generate random bytes array', () => {
+    Math.random = () => 0.1111111111111111;
```

Comment:
We can just move this to `beforeEach` to make it clear. I think we can avoid the use of sinon here.